### PR TITLE
#1784 Remove all global references to config.appName and allow getting this configuration item from content api

### LINKED ISSUE #1784: Make appName come from content-api
### Problem description
Right now appName (what gets appended to the end of page titles) is hard-coded to config. It should be changeable in the content api.

We should also delete the "about" part of the config seeing as it's not used for anything that I can see (- @AlexGilleran ).

### Acceptance Criteria
- As an admin, I can change the `appName` part of the page title
- There's a reasonable default for new instances
- There's a migration script for search.data.gov.au in the config-data-gov-au repo.

### Screenshot / Design / File reference
image

### PR DESCRIPTION
**Pull request by nahidakbar**
### What this PR does

Remove all global references to config.appName and allow getting this configuration item from content api

Fixes #1784

### Checklist

-   [x] There are unit tests to verify my changes are correct or unit tests aren't applicable
-   [x] I've updated CHANGES.md with what I changed.
-   [x] I've linked this PR to an issue in ZenHub (core dev team only)

### CODE REVIEW

File: magda-web-client/src/Pages/HomePage.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
 .homepage-app-container {
     .app-container {
         position: relative;
-        min-height: 100vh;
+        // min-height: 100vh;
```
Comment (AlexGilleran) [inline magda-web-client/src/Pages/HomePage.scss]: Can we delete this?

File: magda-web-client/src/Pages/HomePageComponents/Stories.scss [hunk 2]
```diff
@@ -35,12 +35,16 @@
...
+    }
     .homepage-stories-show-stories-button {
-        position: fixed;
+        //position: fixed;
         border: none;
-        left: 50%;
+        // left: 50%;
         bottom: 0px;
-        margin-left: -67px;
+        // margin-left: -67px;
```
Comment (AlexGilleran) [inline magda-web-client/src/Pages/HomePageComponents/Stories.scss]: Can we delete these?

File: magda-web-client/src/Pages/HomePage.scss (hunk 1 above)
Comment (nahidakbar) [reply magda-web-client/src/Pages/HomePage.scss]: removed!

File: magda-web-client/src/Pages/HomePageComponents/Stories.scss (hunk 2 above)
Comment (nahidakbar) [reply magda-web-client/src/Pages/HomePageComponents/Stories.scss]: removed!

**Pull request merged**